android/app/src/main/java/org/fossasia/openevent/api/DataDownloadManager.java

```diff
@@ -16,7 +26,8 @@
 public final class DataDownloadManager {
     private static DataDownloadManager instance;
 
-    private APIClient client = new APIClient();
+    private APIClient openEventClient = new APIClient();
+    private APIClient graphClient = new APIClient();
```

Comment:
No need for these now. You have created static methods. Just call APIClient.getOpenEventAPI() etc